modules/serverless-kafka-source-kn.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-kafka-source.adoc
+[id="serverless-kafka-source-kn_{context}"]
+= Creating a Kafka event source using the `kn` CLI
+
+This section describes how to create a KafkaSource using the `kn` command.
+
+:FeatureName: Creating a Kafka event source using the `kn` CLI
+include::../modules/technology-preview.adoc[leveloffset=+0]
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, Knative Serving, and the `KnativeKafka` custom resource definitions are installed on your cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have access to a Kafka cluster that produces the Kafka messages you want to import.
+
+.Procedure
+
+. To verify that the Kafka source is working, create a Knative service that dumps incoming messages into the service logs:
++
+[source, terminal]
+----
+$ kn service create event-display \
+    --image quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+----
+
+. Create a `KafkaSource` resource:
++
+[source,terminal]
+----
+$ kn source kafka create mykafkasrc \
+    --servers my-cluster-kafka-bootstrap.kafka.svc:9092 \
+    --topics test-topic --consumergroup test-consumer-group \
+    --sink event-display
+----
++
+The `--servers`, `--topics`, and `--consumergroup` options specify the connection parameters to the Kafka cluster. The `--consumergroup` option is optional.
++
+Additionally, this command makes several assumptions, including:
++
+* `namespace` is `kafka`
+* `pod` is `my-cluster-kafka-0`
+
+. Optional. View details about the created Kafka source:
++
+[source, terminal]
+----
+$ kn source kafka describe mykafkasrc
+Name:              mykafkasrc
+Namespace:         default
```

Comment:
Here the `default` namespace is shown, above the document talks about the `kafka` namespace.

Reply:
Changed `default` to `kafka`. Please let me know if it's incorrect.